docker-contents/README.rst

```diff
@@ -0,0 +1,137 @@
+docker-pinry
+============
+
+.. image:: https://travis-ci.org/pinry/docker-pinry.svg?branch=master
+    :target: https://travis-ci.org/pinry/docker-pinry
+
+A nice and easy way to get a pinry instance up and running using docker. For
+help on getting started with docker see the `official getting started guide`_.
+For more information on Pinry and a demo check out it's `website`_.
+
+
+Getting docker-pinry
+---------------------
+
+Running this will get the latest version of both
+docker-pinry and pinry itself::
+
+  git clone https://github.com/pinry/docker-pinry
+  cd docker-pinry
+  ./bootstrap.sh
+
```

Comment:
Please also help to adapt the document to latest version of our building method.
1. The script path
2. How to config the docker and config file.

Thanks.

Reply:
I've updated the paths in the README.rst